app/api/custom/events.py

```diff
@@ -128,3 +130,47 @@ def reorder_speakers(event_id):
     db.session.commit()
 
     return jsonify({'success': True, 'updates': updates})
+
+
+@events_routes.route('/<string:event_identifier>/reorder-exhibitors', methods=['POST'])
+@to_event_id
+def reorder_exhibitors(event_id):
```

Comment:
Only coorganizer should be able to do this

Reply:
it giving error I was talking about.

Reply:
AssertionError: View function mapping is overwriting an existing endpoint function: events_routes.fwrapper

Reply:
Check the position of decorator. Match with speaker reorder function

Reply:
I added it in same way as speaker reorder then it was giving errror and the I removed that, and error was gone.

Reply:
giving error -
```
@to_event_id
@is_coorganizer
def reorder_exhibitors(event_id):
```

not giving error
```
@to_event_id
def reorder_exhibitors(event_id):
```